exercises/rna-transcription/canonical-data.json

```diff
@@ -1,46 +1,46 @@
 {
   "exercise": "rna-transcription",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "cases": [
     {
       "description": "RNA complement of cytosine is guanine",
```

Comment:
Do the descriptions also need to be updated?

Reply:
I don't think so at this point.  Give #1078 a read.  I am considering closing this PR in favor of making minor changes to metadata.yml and description.md instead (in another PR).  I am currently researching another source to site in metadata.yml.